### DISCUSSION TITLE
Connection with SPSS

### DISCUSSION THREAD

**Discussion by ArhamCMB** [duckdb/duckdb#11910] [General]
Hello! Is it possible to connect DuckDB to SPSS?

I have come across a lot of examples using R & Python, but I dont see much regarding SPSS. I know that SPSS is a bit of a legacy software, but my company uses it, so I am trying to make this connection.

Any pointers to resources / tutorials would be very helpful.

**Comment by dylanmeysmans** [comment]
> Hello! Is it possible to connect DuckDB to SPSS?
> 
> I have come across a lot of examples using R & Python, but I dont see much regarding SPSS. I know that SPSS is a bit of a legacy software, but my company uses it, so I am trying to make this connection.
> 
> Any pointers to resources / tutorials would be very helpful.

@ArhamCMB I managed to implement this as a community extension, see https://github.com/mettekou/duckdb-read-stat.
